package.json

```diff
@@ -22,13 +22,12 @@
     "@changesets/cli": "^2.25.0",
     "@lidofinance/eslint-config": "0.0.2",
     "@next/eslint-plugin-next": "12.1.6",
-    "@rollup/plugin-babel": "5.3.0",
-    "@rollup/plugin-commonjs": "21.0.1",
-    "@rollup/plugin-json": "5.0.1",
+    "@rollup/plugin-babel": "5.3.1",
+    "@rollup/plugin-commonjs": "21.1.0",
+    "@rollup/plugin-json": "5.0.2",
     "@rollup/plugin-node-resolve": "15.0.1",
-    "@types/react": "17.0.19",
-    "@types/react-dom": "17.0.9",
-    "@types/react-is": "17.0.2",
+    "@types/react": "17.0.53",
```

Comment:
why we don't want to use react 18?

Reply:
Because we have react 17 on our widgets and there can be some incompatibilities between 17 and 18. We can migrate all our projects to 18 later